sources/_sector/lending/supply/optimism/_sources.yml

```diff
@@ -21,6 +21,8 @@ sources:
       - name: soSUSD_evt_Redeem
       - name: soUSDC_evt_Mint
       - name: soUSDC_evt_Redeem
+      - name: soUSDCe_evt_Mint
```

Comment:
it appears this decoded table source doesn't exist?
please see CI logs output to find error where query can't find this table